### ISSUE TITLE
How could you save the ouput of the scan in standard format(human readable)

### ISSUE THREAD

**Issue by salooali** [SamJoan/droopescan#56]
Hey, I just wanted to know that how could I store the results? I know by using the redirection operator but with that, you can't save the human-readable output format. kindly let me know if there is any other way...! Thanks

**Comment by SamJoan** [comment]
hi @salooali 

The output section of the readme has some information, did you check that out?

https://github.com/SamJoan/droopescan#output

Let me know if that is enough, or what issues in particular there are to save the human-readable output format. If you want to strip the colors from stdout you can try something like what is proposed here: https://superuser.com/questions/380772/removing-ansi-color-codes-from-text-stream

This is a feature that I have been requested some times in the past so if you want to investigate and do a PR then I'm happy to consider it.

Ta,
Sam

**Comment by salooali** [comment]
Yeah for now I've done a similar thing saving the output without colors. But for making it more readable and presentable 
I would suggest you make its output in HTML report. Like some wrote a tool for wpscan output to convert into HTML
https://github.com/tristanlatr/wpscan_out_parse

**Comment by SamJoan** [comment]
Hi @salooali 

Thank you for your suggestion. At this stage I don't intend to implement this feature because I don't have the free time to do it, although I would have enough time to accept a pull request if contains unit tests and abides by the collaboration guidelines in the readme. 

In the meantime you can convert the cli output to HTML using aha. https://stackoverflow.com/questions/245121/a-library-to-convert-ansi-escapes-terminal-formatting-color-codes-to-html/2975843#2975843

Thanks!
Sam

**Comment by salooali** [comment]
Hi @SamJoan 
Thank you for your suggested tool for now It's really very helpful for me.
Sure I'll try to resolve this issue in my free time.
